utils/memset_safe.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __UTILS_MEMSET_SAFE_H__
+#define __UTILS_MEMSET_SAFE_H__
+
+#include "common/errors.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef __STDC_LIB_EXT1__
+
+retcode_t memset_safe(void *dest, size_t destsz, int ch, size_t count);
+
+#else
+
```

Comment:
Does this need to be added?
```suggestion
#define __STDC_WANT_LIB_EXT1__ 1
```

According to https://en.cppreference.com/w/c/string/byte/memset:
> As with all bounds-checked functions, memset_s is only guaranteed to be available if `__STDC_LIB_EXT1__` is defined by the implementation and if the user defines `__STDC_WANT_LIB_EXT1__` to the integer constant 1 before including string.h.

Reply:
thank you for advice